src/components/AvatarCropModal/AvatarCropModal.js

```diff
@@ -77,6 +77,7 @@ const AvatarCropModal = (props) => {
     const rotation = useSharedValue(0);
     const translateSlider = useSharedValue(0);
     const isPressableEnabled = useSharedValue(true);
+    const isLoading = useSharedValue(false);
```

Comment:
Can we add a comment for `isLoading` since this is a very generic name so having a comment would make it easier to understand the use case.